src/scss/layouts/WordsOverview.scss

```diff
@@ -8,28 +6,71 @@
 
     .main_container--grid {
         margin: 30px;
+        margin-top: 0;
+        min-width: 80%;
     }
 
-        .wordBorder {
-            display: grid;
-            grid-template: 100% / repeat(3, 1fr) 1fr;
-            border: 2px solid rgb(168, 166, 163);
+    .wordBorder {
+        display: grid;
+        grid-template-columns: 1.5fr 0.5fr 1.2fr 1fr;
+        grid-template-rows: 29px;
+        border: 2px solid rgb(168, 166, 163);
+        margin: 10px;
+        padding-left: 20px;
+        box-shadow: $box-shadow;
+        background-color: $container-color;
+        height: 30px;
+
+        .transWorsDe,
+        .transWordCz {
+            border: 1px solid rgb(168, 166, 163);
+            width: auto;
+            overflow: hidden;
+            white-space: nowrap;
+            text-overflow: ellipsis;
+        }
+
+        p.transWorsDe,
+        p.transWordCz {
+            color: #262548;
+            font-size: 14px;
+            font-weight: 600;
+            text-align: center;
+            margin: 5px;
+            padding: 1px;
+            align-self: center;
+        }
+
+        .trans {
+            display: flex;
+            justify-self: flex-end;
+            align-self: center;
+            margin-right: 10px;
+            font-size: 12px
+        }
+
+        .buttonsSection {
+            display: flex;
             justify-content: center;
             align-items: center;
-            margin: 15px;
-            box-shadow: $box-shadow;
-            background-color: $container-color;
-
-            p {
-                margin: 5px;
-                padding: 2px;
-            }
 
             .deleteButton {
                 background-color: $button-color;
                 border: none;
                 color: #fff;
-                border-radius: 8px;
+                border-radius: 3px;
+                padding: 5px;
+                cursor: pointer;
+                height: 22px;
```

Comment:
Zkus to udělat bez použití pevného nastavení width a height. Tak jak jsem už psal v jiném komentu, widht a height by měla určovat velikost inside elementů.

Reply:
upraveno na procenta